packages/just-scripts/src/tasks/sassTask.ts

```diff
@@ -61,7 +60,23 @@ export function sassTask(
                   const css = result.css.toString();
 
                   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                  postcss([autoprefixerFn, ...postcssPlugins!])
+                  const plugins = [autoprefixerFn, ...postcssPlugins!];
+
+                  // If the rtl plugin exists, insert it after autoprefix.
+                  if (postcssRtl) {
+                    plugins.splice(1, 0, postcssRtl({}));
+                  } else {
+                    logger.warn('Please install missing dependency "postcss-rtl" plugin so that directional styles will flip.');
```

Comment:
I think we should move this to logger.info and instead of using words like missing, provide a suggestion:

```suggestion
                    logger.info('Suggestion: installing the dependency "postcss-rtl" plugin allows directional styles will flip.');
```

Reply:
Fixed!